requirements/windows.txt

```diff
@@ -1,2 +1,3 @@
 ipykernel
-cloudpickle
\ No newline at end of file
+cloudpickle
+pip>=19.3.1
```

Comment:
I think this approach is fine for now, let's add a comment here to explain why we need the pin. Then we are good to go!